src/test/java/com/pszymczyk/IntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package com.pszymczyk;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class IntegrationTest {
+
+    @Autowired
+    TestRestTemplate testRestTemplate;
```

Comment:
Czy te pola nie powinny byc prywatne?

Reply:
Nie widzę żadnej zalety, to testy ale jak wolisz mogę zrobić `private`